modules/jaeger-deploy-streaming.adoc

```diff
@@ -10,6 +10,11 @@ The `streaming` deployment strategy is intended for production environments, whe
 
 The `streaming` strategy provides a streaming capability that sits between the collector and the storage (Elasticsearch). This reduces the pressure on the storage under high load situations, and enables other trace post-processing capabilities to tap into the real time span data directly from the streaming platform (Kafka).
 
+[NOTE]
+====
+Note that the streaming strategy requires an additional Red Hat license for AMQ Streams. If you do not have an AMQ Streams subscription, contact your sales representative for more information.
```

Comment:
When rendered in md, it shows a Note box with the first word in the description 'Note'. When rendered in ocp docs not sure if it would have the same duplicate word, or whether the first Note is replaced by a symbol?

Same comment about license -> subscription?

Reply:
updated.